src/CitMovie.Data/Repositories/ILanguageRepository.cs

```diff
@@ -0,0 +1,13 @@
+using CitMovie.Models.DomainObjects;
+
+namespace CitMovie.Data;
+
+public interface ILanguageRepository
+{
+    Task<Language?> GetLanguageByIdAsync(int languageId);
+    Task<IList<Language>> GetLanguagesAsync( int page, int pageSize);
+    Task <Language> CreateLanguageAsync(Language language);
+    Task <Language> UpdateLanguageAsync(Language language);
+    Task <Language>DeleteLanguageAsync(int languageId);
```

Comment:
Should we just delete these if we don't use them in the code?

Reply:
Sure we can